### PR TITLE
Unxz images if ending with xz format

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -59,11 +59,23 @@
...
-  with_items: "{{ os_images_list }}"
+- block:
+    - name: Download the image to the directory when an image_url is defined
+      get_url:
+        url: "{{ item.image_url }}"
+        dest: "{{ image_dest }}"
+      with_items: "{{ os_images_list }}"
+    - debug: var=image_dest
+      with_items: "{{ os_images_list }}"
+    - debug: var=image_is_xz
```
Comment (jovial) [inline tasks/main.yml]: Did you mean to leave the debug statements?

Comment (brtkwr) [reply tasks/main.yml]: No sir, definitely not, thanks!

**Comment by jovial** [comment]
Should we add either install the `xz` package or list it as a dependency in the README?

**Comment by brtkwr** [comment]
> Should we add either install the `xz` package or list it as a dependency in the README?

Good point, I've made it install xz :)

**Review by jovial — APPROVED** [review]
Looks good to me.

**Pull request merged**